test-case/manual/Audio_privacy.md

```diff
@@ -0,0 +1,48 @@
+# Verify Audio Privacy (Hard Mute) Functionality
+Verify Audio Privacy (Hard Mute) Functionality in Linux
+
+## Preconditions
+1. The RVP should have active audio endpoints.
+2. The RVP should have a functional hard mute switch or toggle.
+3. Enable BIOS settings:
+    3a. Microphone Privacy Mode = "HW managed Microphone Privacy"
+    3b. Enable corresponding codec options, e.g., SNDW#3 [x] for SoundWire.
+
+## Test Description
+* Verification of audio privacy (hard mute) functionality in a Linux system.
+* Should not capture audio samples when the audio privacy switch is turned ON.
+
+## Steps to Execute
+1. Turn ON the audio privacy switch. Refer to RVP TOPS documents to find the switch location.
+    Example: Switch identification mentioned as "MIC_privacy_SW_IN".
+2. Capture audio via 3.5mm jack ports:
```

Comment:
I think we need to cover both Jack and DMIC.

Reply:
okay i wil add and update you